mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -251,15 +251,15 @@ static void populateDefaults() {
   }
 
   auto getOutputDim = [](int64_t inputLen, int64_t filLen, int padLen,
-                         int strideLen) {
-    return (inputLen + 2 * padLen - filLen) / strideLen + 1;
+                         int strideLen, int dilLen) {
+    return (inputLen + 2 * padLen - (filLen - 1) * dilLen - 1) / strideLen + 1;
```

Comment:
Neat! Thanks for figuring this out. I think it is a most recent bug I introduced where I didn't consider about dilation in coming out with output lengths. Is there a good reference that this equation got explained?

Does this fix https://github.com/ROCmSoftwarePlatform/llvm-project-private/issues/41? If so, could you also help to update that list?

Reply:
This is how MIOpen calculates output height/width. Chao or Jing mentioned it before in MLIR MIOpen dialect. And it was the same as the equation I came up with. 

The issue#41 is updated automatically.

Reply:
Could you remove the fixed test cases that were fixed by this? I didn't see the issue changed most recently.